src/main/java/A2019/Day6/Ex1.java

```diff
@@ -0,0 +1,54 @@
+package A2019.Day6;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ex1 {
+    public static void main(String[] args) throws IOException {
+        List<Planet> planets = new ArrayList<>();
+        String line;
+        try (BufferedReader br = new BufferedReader(new FileReader("src/main/java/A2019/Day6/input.txt"))) {
+            while ((line = br.readLine()) != null) {
+                String[] tabPlanet = line.split("\\)");
+                Planet planet1 = new Planet(tabPlanet[0]);
+                Planet planet2 = new Planet(tabPlanet[1]);
+                planet2.setOrbit(planet1);
+                if (planets.contains(planet1)) {
+                    planet1 = planets.get(planets.indexOf(planet1));
+                    planet2.setOrbit(planet1);
+                } else {
+                    planets.add(planet1);
+                }
+                if (planets.contains(planet2)) {
+                    planets.set(planets.indexOf(planet2),planet2);
+                    for (Planet p : planets) {
+                        if(p.getOrbit() != null && p.getOrbit().equals(planet2)) {
+                            p.setOrbit(planet2);
+                        }
+                    }
```

Comment:
Il serait peut-être plus simple de récupérer les planètes qui t'intéressent, puis de les mettre à jour ? 🪕 
```suggestion
                    planets.stream()
                            .filter(...)
                            .forEach(...);
```